doc/requirements.txt

```diff
@@ -0,0 +1,117 @@
+
+    `NumPy <http://numpy.scipy.org/>`_ >= 1.7.1 < MAX_VERSION
+        Earlier versions could work, but we don’t test it.
+
+    `SciPy <http://scipy.org>`_ >= 0.11 < MAX_VERSION
+        Only currently required for sparse matrix and special functions support, but highly recommended. SciPy >=0.8 could work, but earlier versions have known bugs with sparse matrices.
+
+    `BLAS`_ installation (with Level 3 functionality)
+        Recommended: MKL, which is free through Conda. Otherwise, we suggest to install OpenBLAS, with the development headers (``-dev``, ``-devel``, depending on your Linux distribution). 
+
+.. _BLAS: http://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms
+.. _Python: http://www.python.org/
+
+(Optional)
+
+    ``g++`` >= 4.2
+    ``python-dev``
+        **Highly recommended.** Theano can fall back on a NumPy-based Python execution model, but a C compiler allows for vastly faster execution.
+
+    `nose <http://somethingaboutorange.com/mrl/projects/nose/>`_ >= 1.3.0
+        Recommended, to run Theano's test-suite.
+
+    `Sphinx <http://sphinx.pocoo.org/>`_ >= 0.5.1, `pygments <http://pygments.org/>`_
+        For building the documentation. LaTeX_ and dvipng_ are also necessary for math to show up as images.
+
+    `pydot-ng <https://github.com/pydot/pydot-ng>`
+        To handle large picture for gif/images.
+
+    `NVIDIA CUDA drivers and SDK`_
+        Required for GPU code generation/execution on NVIDIA gpus. See instruction below.
+
+    `libgpuarray`_
+        Required for GPU/CPU code generation on CUDA and OpenCL devices (see: :ref:`gpuarray`.)
+
+Conda
+-----
+
+You can install most of these packages with your system package manager or compiling them manually, 
+but we ONLY support the installation of the requirements through conda. 
+
+1. If you want fast compiled code (recommended), make sure you have g++ (Windows/Linux) or Clang (OS X) installed.
+
+2. Install Miniconda_.
+
+3. Install mandatory requirements: 
+.. code::
+
+    conda install numpy, scipy, mkl
+
+4. Install the optional requirements available through Conda:
+
+.. code::
+
+    conda install nose, Sphinx, pydot-ng
+
+Go to the next section to install the GPU backends.
+
+
+.. _GPU:
+
+GPU
+---
+
+1. Install NVIDIA CUDA drivers and SDK (optional)
+
+
+    `NVIDIA CUDA drivers and SDK`_
+        Required for GPU code generation/execution on NVIDIA gpus
+
+    `libgpuarray`_
+        Required for GPU/CPU code generation on CUDA and OpenCL devices (see: :ref:`gpuarray`.)
+
+        .. note:: 
+
+            OpenCL support is still minimal for now.
+
+        .. note:: 
+        
+            libgpuarray is the new backend and is not required for the old version.
+
+The first thing you'll need for Theano to use your GPU is Nvidia's
```

Comment:
Transform this verbose instruction into a **list** of operations.
I expect something like:

- Install CUDA
    1. Download CUDA
    2. ...
- Install libgpuarray
    1. Download libgpuarray
    2. ...